tests/sql/functions_test.py

```diff
@@ -256,3 +258,70 @@ def test_floor(types, expected) -> None:
             )
             == expected
         )
+
+
+def test_aggregate(session: Session):
+    """
+    Test the `aggregate` Spark function
+    """
+    query = parse(
+        """
+    select
+      aggregate(items, '', (acc, x) -> (case
+        when acc = '' then element_at(split(x, '::'), 1)
+        when acc = 'a' then acc
+        else element_at(split(x, '::'), 1) end)) as item
+    from (
+      select 1 as id, ARRAY('b', 'c', 'a', 'x', 'g', 'z') AS items
+    )
+    """,
+    )
+    exc = DJException()
+    ctx = ast.CompileContext(session=session, exception=exc)
```

Comment:
Should these tests also include an assert that the `exc` variable doesn't contain any errors?

Reply:
Good point, added!